cms/core/tests.py

```diff
@@ -1,9 +1,36 @@
+from xml.dom import ValidationErr
```

Comment:
This looks like an errant import from Code

Reply:
... yeah, this is completely unhelpful. Need to discover what's causing this...